tools/AzureRM.BootStrapper/AzureRM.BootStrapper.Tests.ps1

```diff
@@ -1291,3 +1265,132 @@ Describe "Set-BootstrapRepo" {
         }
     }
 }
+
+Describe "Set-AzureRmDefaultProfile" {
+    InModuleScope AzureRM.Bootstrapper {
+        $sb = {
```

Comment:
Should add tests for:

(1) Setting a default profile twice (verify that file is not altered twice